src/deroff.rs

```diff
@@ -1060,23 +1164,56 @@ fn test_comment() {
     assert_eq!(deroffer.s, "\nworld".to_owned());
 }
 
-fn deroff_files(files: &[String]) -> std::io::Result<()> {
+fn deroff_files<P: AsRef<std::path::Path>, S: ToString>(
```

Comment:
We would need another binary for this, looks like we were missing this. @scooter-dangle 

To do this, we have to create `src/bin/deroff.rs`.

Reply:
Either that or add an option to the top-level binary that just deroffs files. Probably makes sense to have a binary that just deroffs?

Either way, will make it easier for us to test the deroffing against the contents of https://github.com/rust-dc/manpages.

Reply:
@scooter-dangle @pickfire I'll work on this tonight and make a pr

Reply:
Interesting, I didn't know @kbknapp thought about this so fast. He even have a test repository prepared.

Ideally, not a top-level binary, just a binary for deroff like the original python `deroff.py`, I don't think it is useful for users to run this.

@charliethomson Take your time, we can fight the final boss slower. I wonder if would be cool if everyone can fight it together, like in a meetup.